### PR TITLE
Apify project structure proposal #84

### PR DESCRIPTION
**Pull request by drobnikj**
Apify project structure proposal

### CODE REVIEW

File: src/templates/apify_project/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "description": "This is a boilerplate of an Apify actor.",
+    "dependencies": {
+        "apify": "^0.15.2"
+    },
+    "devDependencies": {
+        "@apify/eslint-config": "0.0.3",
+        "eslint": "^6.4.0"
+    },
+    "scripts": {
+        "start": "node src/main.js",
```
Comment (mtrunkat) [inline src/templates/apify_project/package.json]: I would also add

```
        "lint": "./node_modules/.bin/eslint ./src ./test --ext .js,.jsx",
        "lint:fix": "./node_modules/.bin/eslint ./src ./test --ext .js,.jsx --fix"
```

File: src/lib/consts.js [hunk 2]
```diff
@@ -60,6 +60,16 @@ exports.ACTS_TEMPLATES = {
         },
         skipOptionalDeps: true,
     },
+    apify_project: {
+        name: 'Apify project - Template for Apify projects, mainly for Apify Marketplace developers',
```
Comment (mtrunkat) [inline src/lib/consts.js]: What about: `Template containing boilerplate and code style rules used for Apify company internal projects.`

Comment (jancurn) [reply src/lib/consts.js]: or even better `Standardized template containing boilerplate and code style rules used for Apify Marketplace projects`

Comment (jancurn) [reply src/lib/consts.js]: This way marketplace developer will know they should use it

**Pull request merged**